files/his-dev/docker-compose.yml

```diff
@@ -202,9 +196,9 @@ services:
       API_URL: /openmrs
       SPA_CONFIG_URLS: ${SPA_CONFIG_URLS}
       SPA_DEFAULT_LOCALE: ${SPA_DEFAULT_LOCALE}
-      ODOO_PUBLIC_URL: https://${ODOO_HOSTNAME:-http://localhost:8069}
-      OPENMRS_PUBLIC_URL: https://${O3_HOSTNAME:-http://localhost}
-      SENAITE_PUBLIC_URL: https://${SENAITE_HOSTNAME:-http://localhost:8081}
+      ODOO_PUBLIC_URL: ${SERVER_SCHEME}://${ODOO_HOSTNAME:-localhost:8069}
+      OPENMRS_PUBLIC_URL: ${SERVER_SCHEME}://${O3_HOSTNAME:-localhost}
```

Comment:
This means we need to update `.env`; `SERVER_SCHEME=https`

Reply:
We must inform @ibacher  to make this change once this is deployed.

Reply:
That will work. However, I would like that we maintain consistency with what we have in `ozone-docker-compose` project where we are defaulting to `http` via `.env` file (see: https://github.com/ozone-his/ozone-docker-compose/blob/main/.env#L10).